.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup
-      - run: npx yarn-deduplicate --strategy=highest --list --fail
+      - run: yarn yarn-deduplicate --strategy=highest --list --fail
```

Comment:
nit: `yarn deduplicate --list --fail` instead to share the strategy with `deduplicate` script?

Reply:
the list and fail is for the build to error